scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class CommentsController < ApplicationController
+
+  def index
+  end
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @post.comments.create!(comment_params.merge(user: current_user))
+    redirect_to post_path(@post, @user), notice:
+    "#{current_user.email} has left a new comment!"
```

Comment:
I like how you broke this onto a separate line to keep the line length below 80, but I'd break it a bit differently according to common ruby style:

```ruby
redirect_to post_path(@post, @user), 
  notice: "#{current_user.email} has left a new comment!"
```

notice the break after a comma, and the indentation to indicate the lines are really 1